src/v2/view-builder/views/consent/ConsentView.js

```diff
@@ -0,0 +1,39 @@
+import { loc } from 'okta';
+import BaseView from '../../internals/BaseView';
+import BaseForm from '../../internals/BaseForm';
+import ConsentViewHeader from './ConsentViewHeader';
+
+const Body = BaseForm.extend(
+  {
+    noButtonBar: false,
+    noCancelButton: false,
+    save: () => loc('consent.required.consentButton', 'login'),
+    cancel: () => loc('consent.required.cancelButton', 'login'),
+    title: false,
+  },
+);
+
+export default BaseView.extend({
+  className: 'admin-consent-required',
```

Comment:
can we leverage `ViewClassNamesFactory` ?

Reply:
What's the reasoning behind separating the className from the View?